static-build/index.tsx

```diff
@@ -28,15 +28,38 @@ const toOutput: Output = {
   'faqs/index.html': renderPage(<FAQPage />),
 };
 
+function getPrevOrNextData(testEntry: [string, Test]): PaginationData {
+  return {
+    link: './../' + testEntry[0] + '/',
+    meta: {
+      title: testEntry[1].meta.title,
+      shortDesc: testEntry[1].meta.shortDesc,
+    },
+  };
+}
+
 function addTestPages(tests: Tests, basePath = '') {
-  for (const [testPath, test] of Object.entries(tests)) {
+  const testEntries: [string, Test][] = Object.entries(tests);
+  const len = testEntries.length;
+  testEntries.forEach(([testPath, test], i) => {
     const testBasePath = basePath + testPath + '/';
+
+    const prevData: PaginationData =
+      i !== 0
+        ? getPrevOrNextData(testEntries[i - 1])
+        : getPrevOrNextData(testEntries[len - 1]);
+
+    const nextData: PaginationData =
```

Comment:
not sure how to handle it yet, but sometimes next and prev point to the same page and create duplicate links:

![Screen Shot 2022-01-13 at 3 20 10 PM](https://user-images.githubusercontent.com/1134620/149424165-f14ed41c-ab4f-4a49-af67-b08e6a413aa0.png)
 
2 ideas for a solution
1. have it be a link to the next/prev section collection landing page?
2. hide one of them depending if at the beginning or end of a collection
3. your idea here?

Reply:
What if we have both 1 and 2.
You can check [this](https://github.com/deepanshu44/tooling.report/tree/nav-test) where I developed a (sort of) prototype.

<details>
<summary> When at the beginning </summary>

![goto-parent](https://user-images.githubusercontent.com/62979448/149516863-21b474a9-dece-46da-9418-d834798e10b4.jpg)
</details>
<details>
<summary> When at the end </summary>

![Screenshot_end](https://user-images.githubusercontent.com/62979448/149517071-bb05059f-76de-4c1c-8312-18d0d95308f9.jpg)



</details>

Reply:
@argyleink 
What if we have both 1 and 2.
You can check [this](https://github.com/deepanshu44/tooling.report/tree/nav-test) where I developed a (sort of) prototype.

<details>
<summary> When at the beginning </summary>

![begining](https://user-images.githubusercontent.com/62979448/149517756-e7755063-968e-4605-9e19-e94c9666cfdc.jpg)


</details>
<details>
<summary> When at the end </summary>

![end](https://user-images.githubusercontent.com/62979448/149517618-3d5118c8-c537-490c-a481-962f8689d6c2.jpg)
</details>

Reply:
nice, having both feels well rounded 🙂